generators/app/templates/src/config/navigation.ejs

```diff
@@ -0,0 +1,77 @@
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import React from 'react';
+import { Button } from 'react-native';
+
+<%_ } _%>
+import * as Routes from '../constants/routes';
+import { blue, white } from '../constants/colors';
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import { actionCreators as drawerActions } from '../redux/drawer/actions';
+<%_ } _%>
+
+import I18n from './i18n';
+
+// Default nav options for all screens
+const defaultNavOptions = ({ navigation }) => ({
+  // Change screen title from i18n traslates files
+  title: I18n.t(`app:${navigation.state.routeName}`),
```

Comment:
I'd use `{ strings }` from its `i18n` since this might not be defined

Reply:
I can't use `{ strings }` pattern because the translations is translated dynamically according `routeName` parameter.